lib/tasks/import_facilities.rake

```diff
@@ -0,0 +1,9 @@
+require "csv"
+
+task create_facilities: :environment do
```

Comment:
All the rake commands work great, sans the extra error message the `db:create` has. A future suggestion would be to have a rake command that runs all four setup commands to streamline the setup.